src/views/networkpolicies/new/components/NetworkPolicyFormActionButtons.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+import { useNavigate } from 'react-router-dom-v5-compat';
+
+import { ActionGroup, Button } from '@patternfly/react-core';
+import { useNetworkingTranslation } from '@utils/hooks/useNetworkingTranslation';
+
+type NetworkPolicyFormActionButtonsProps = {
+  isDisabled: boolean;
+  isLoading: boolean;
+};
+
+const NetworkPolicyFormActionButtons: FC<NetworkPolicyFormActionButtonsProps> = ({
```

Comment:
Can you extract this component to a shared location? looks like we can use this in multiple locations :)

Reply:
only 1 location, its the create/cancel button

Reply:
We can also use it in nad form, next PR :)